CodeGeneration/Sources/SyntaxSupport/gyb_generated/DeclNodes.swift

```diff
@@ -42,9 +42,12 @@ public let DECL_NODES: [Node] = [
                isOptional: true,
                collectionElementName: "Modifier"),
          Child(name: "TypealiasKeyword",
-               kind: "TypealiasToken",
+               kind: "KeywordToken",
                tokenChoices: [
```

Comment:
Why does the swift-generation add `tokenChoices` for all these? (The corresponding Python doesn't define any, which makes more sense to me)

Reply:
I’m not entirely sure but I’ve got changes locally that refactor `tokenChocies` and `textChoices` on the CodeGeneration side, so I don’t care right now.